pvm/ts/src/rpc.ts

```
@@ -342,6 +374,11 @@ export class RPC {
                     const _action = action as PlayerActionType | NonPlayerActionType;
                     const command = new PerformActionCommandWithResult(from, to, Number(index), BigInt(amount || "0"), _action, Number(nonce), validatorPrivateKey, data);
                     result = await command.execute();
+                    
+                    // Broadcast game state update to all players at this table
+                    if (result && to) {
+                        await this.broadcastGameStateUpdate(to, from);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)